FittableFontLabel/FittableFontLabel iOS Tests/FittableFontLabelTests.swift

```diff
@@ -24,6 +24,8 @@ class FittableFontLabelTests: XCTestCase {
     let shortText = "Etiam turpis elit"
     // swiftlint:disable:next line_length
     let longText = "Etiam turpis elit, efficitur nec suscipit vitae, aliquam vitae lorem. Aenean mauris tortor, semper vel facilisis nec, facilisis ut neque. Aliquam erat volutpat. Nam eget varius mauris, ut condimentum quam. Duis ex nisi, iaculis in ex ut, lobortis pretium est. Sed non lorem venenatis, fermentum quam aliquet, ultrices arcu. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Duis ullamcorper convallis tellus, vitae bibendum mi dapibus in. Morbi pellentesque aliquet urna, id aliquet orci efficitur sed. Donec et mi urna. Proin sagittis semper massa in semper."
+
+  let optimalFontSize: CGFloat = 2.0703125
```

Comment:
It has been removed in the latest commit. I have now fixed the indentation of the subsequent commit as well.